packages/payload/src/fields/config/client.ts

```diff
@@ -300,6 +300,12 @@ export const createClientField = ({
                 i18n,
                 importMap,
               })
+            } else if (key === 'label' || key === 'description') {
```

Comment:
Couldn't this be simplified?
```
 } else if ((key === 'label' || key === 'description') && typeof tab[key] === 'function') {
  clientTab[key] = content({ t: i18n.t })
} else {
  clientTab[key] = tab[key]
}
```

Reply:
You're absolutely right. Tunnel vision. I opted to put `tab[key]` into it's own variable since I didn't like either of:

```
// Concise but readability is poor
if (...) {
  clientTab[key] = tab[key]({ t: i18n.t })
}
```

Or

```
// More verbose and we don't reuse content anywhere
if (...) {
  const content = tab[key]
  clientTab[key] = content({ t: i18n.t })
}
```